tools/ports/__init__.py

```diff
@@ -412,7 +412,10 @@ def handle_use_port_error(arg, message):
   utils.exit_with_error(f'Error with `--use-port={arg}` | {message}')
 
 
-def handle_use_port_arg(settings, arg):
+def handle_use_port_arg(settings, arg, error_handler=None):
+  if not error_handler:
+    def error_handler(message):
```

Comment:
Does this overriding of the variable via a function definition really work?

It would make more sense to me if there was a direct assignment to error handler here. e.g. `error_handler = lamdba message:  handle_use_port_error(arg, message)`

Reply:
This was my first implementation... but unfortunately the lambda syntax triggers flake8 to fail with this error message:

```
./tools/ports/__init__.py:413:5: E731 do not assign a lambda expression, use a def
    error_handler = lambda message: utils.exit_with_error(f'Error with `--use-port={arg}` | {message}')
```

I am not sure that makes any sense to me because the lambda expression is so much more readable that defining an inner function. If you know a way around this, then let me know...

Reply:
Fair enough.. if `def` works here I guess that is fine.